### PR TITLE
built-ins/Promise/*: make all indentation consistent (depth & character)

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: test/built-ins/Promise/prototype/finally/this-value-non-promise.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ features: [Promise.prototype.finally]
 var called = false;
 var p = new Proxy(Promise.resolve(), {});
 var oldThen = Promise.prototype.then;
-Promise.prototype.then = () => { called = true; };
+Promise.prototype.then = () => {
```
Comment (ljharb) [inline test/built-ins/Promise/prototype/finally/this-value-non-promise.js]: Oops, this probably shouldn’t have been an arrow function.

File: test/built-ins/Promise/prototype/then/rxn-handler-rejected-next.js [hunk 2]
```diff
@@ -21,33 +21,33 @@ var promise = new Promise(function(_, reject) {
...
-        'Expected each "onFulfilled" handler to be invoked exactly once in series. ' +
-        'Expected: abc. Actual: ' + log
-      );
-      return;
-    }
+  $DONE('This promise should not be fulfilled.');
+}, function() {
+  if (log !== 'abc') {
+    $DONE(
+      'Expected each "onFulfilled" handler to be invoked exactly once in series. ' +
```
Comment (ljharb) [inline test/built-ins/Promise/prototype/then/rxn-handler-rejected-next.js]: This error message is incorrect; onFulfilled should be onRejected.

File: test/built-ins/Promise/prototype/finally/this-value-non-promise.js (hunk 1 above)
Comment (leobalter) [reply test/built-ins/Promise/prototype/finally/this-value-non-promise.js]: surely not, but also not really a thing to be fixed on this PR. After this work I should take a look for all the occurrences of arrow fns in the built-ins folder.

File: test/built-ins/Promise/prototype/then/rxn-handler-rejected-next.js (hunk 2 above)
Comment (leobalter) [reply test/built-ins/Promise/prototype/then/rxn-handler-rejected-next.js]: another one to be fixed in a separate PR.

**Pull request merged**